UnityProject/Assets/ScriptableObjects/Learning/Tips.asset

```diff
@@ -0,0 +1,131 @@
+  - There are four types of damage your character can receive; brute, burn, toxin
+    and oxygen. All of them are equally deadly but have different effects on you.
+  - Boxes automatically pick minnerals from the ground when pulled.
+  - Never take pills labaled as "Happy Pills".
+  - There are over a hunrded recipes for drinks and food. Try and learn them all.
+  - ProtoKineticAccelators (PKA) can destroy walls.
+  - A crewmate who's competent enough to do their job is an antagonist's worst nightmare.
+  - Remember to do your paperwork before requesting something from the <color=blue>Head
+    of Personnel (HoP)</color>
+  - Preform CPR to revive a dying crewmate by changing your intent to <color=green>Help</color>
+    then targeting the best.
+  - Snacks from vending machines aren't fuffling, but they're good enough to keep
+    you running for a couple more minutes.
+  - Say <i>"Come Ian!"</i> near Ian the dog to make him come to you.
+  - Drinking alchole too much gives you liver damage. Which means an annoying visit
+    to medbay if you want to continue living.
+  - Your character's age has minimal effects on gameplay. Don't worry about being
+    too young or too old.
+  - Buckle people and yourself into chairs by dragging their character to the chair.
+  - A handcuffed or restrainted player cannot move freely if you grab or pull them.
+  - Cat ears change the way you speak. Like Thwise, Mwaster~ uwu.
+  - Customization options don't end in the character creator. Visit a vending machine
+    that sells costumes, outfits and accesories to make your character look the way
+    you'd love.
+  - You can be sucked into space if you're not careful enough with your steps and
+    where you go.
+  - If you see purple glitter spreading across the hallway or a room. Run.
+  - Never trust monkeys.
+  - Attacking someone for no reason is the fastest way to get banned off a server.
+  - If you die, there's always a chance you can respawn as another character or play
+    as a "Ghost Role".
+  - "You can restore your hygiene by washing your hands in a sink or taking a shower
+    \u2014 click the showerhead to turn it off and stand under it for a bit."
+  - Some computers and doors require an access level to be used.
+  - The blob is a jelly-like organism that rapidly spreads across the station, destroying
+    obstacles such as walls, windows, and crew members. At the start of the round,
+    one or several player-controlled blobs will deploy somewhere on the station and
+    immediately begin spreading outwards from that point. It has a strong vulnerability
+    to fire - welders, flamethrowers and actual fire will damage and destroy the
+    blob, a tile at a time.
+  - Extended is a gamemode where there are no antagonists selected, giving players
+    the ability to experiment with station systems and roleplay in peace for as long
+    as they want.
+  - AIs can malfunction and rebel against humans.
```

Comment:
Should we be including tips that reference something that isn't implemented?
```suggestion
  - Eventually we'll have AIs that can malfunction and rebel against humans.
```

Reply:
Someone will forget updating this in the future lol